bundle/tests/environments_job_and_pipeline/databricks.yml

```diff
@@ -0,0 +1,44 @@
+resources:
+  pipelines:
+    nyc_taxi_pipeline:
+      name: "nyc taxi loader"
+      libraries:
+        - notebook:
+            path: ./dlt/nyc_taxi_loader
+
+environments:
+  development:
+    mode: development
+    resources:
+      pipelines:
+        nyc_taxi_pipeline:
+          target: nyc_taxi_development
+          development: true
+
+  staging:
+    resources:
+      pipelines:
+        nyc_taxi_pipeline:
+          target: nyc_taxi_staging
+          development: false
+
+  production:
+    mode: production
+    resources:
+      pipelines:
+        nyc_taxi_pipeline:
+          target: nyc_taxi_production
+          development: false
+          photon: true
+
+      jobs:
+        pipeline_schedule:
+          name: Daily refresh of production pipeline
+
+          schedule:
+            quartz_cron_expression: 6 6 11 * * ?
+            timezone_id: UTC
+
+          tasks:
+            - pipeline_task:
+                pipeline_id: "to be interpolated"
```

Comment:
Why are these file adds? Weren't they already part of the tree?

Reply:
These tests are left intentionally, we have 2 sets of same tests: one that have `targets` section and one `environments`. These were tests which failed after rename with no backward compatibility in place, so it helps to be extra safe and ensure we're not breaking existing configs

Reply:
Makes sense, thanks.